blackbird_python/blackbird/listener.py

```diff
@@ -278,7 +276,7 @@ def exitExpressionvar(self, ctx: blackbirdParser.ExpressionvarContext):
         try:
             # assume all variables are scalar
             final_value = PYTHON_TYPES[vartype](value)
-        except:
+        except TypeError:
```

Comment:
good catch!

Reply:
I wonder why I originally had this as a general except... perhaps there are some edge cases where something other than a `TypeError` can be raised?

Reply:
I was thinking about this as well, but couldn't figure out any other cases. Either way, I think it's better to add these cases separately if they show up, instead of having a catch-all except statement.

Reply:
agreed